<ui version="4.0" >
 <class>DlgPrefMidiDeviceDlg</class>
 <widget class="QWidget" name="DlgPrefMidiDeviceDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="minimumSize" >
      <size>
       <width>100</width>
       <height>50</height>
      </size>
     </property>
     <property name="title" >
      <string>MIDI Devices</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QListWidget" name="listMidiDevices" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize" >
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="verticalLayout2" >
     <item>
      <widget class="QGroupBox" name="groupBox_1" >
       <property name="title" >
        <string>Device Information</string>
       </property>
       <widget class="QLabel" name="label_5" >
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>30</y>
          <width>51</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text" >
         <string>Name:</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelDeviceName" >
        <property name="geometry" >
         <rect>
          <x>60</x>
          <y>30</y>
          <width>231</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="labelDeviceStatus" >
        <property name="geometry" >
         <rect>
          <x>60</x>
          <y>60</y>
          <width>231</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_6" >
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>60</y>
          <width>51</width>
          <height>18</height>
         </rect>
        </property>
        <property name="maximumSize" >
         <size>
          <width>61</width>
          <height>18</height>
         </size>
        </property>
        <property name="text" >
         <string>Status:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7" >
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>90</y>
          <width>111</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text" >
         <string>Preset Available:</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelPresetAvailable" >
        <property name="geometry" >
         <rect>
          <x>120</x>
          <y>90</y>
          <width>171</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="btnLoadDefaults" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="geometry" >
         <rect>
          <x>20</x>
          <y>120</y>
          <width>200</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text" >
         <string>Load/Import Default Bindings</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="minimumSize" >
        <size>
         <width>100</width>
         <height>150</height>
        </size>
       </property>
       <property name="title" >
        <string>Device Options</string>
       </property>
       <widget class="QLabel" name="label_4" >
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>60</y>
          <width>81</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text" >
         <string>MIDI Output:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboMidiTxEnable" >
        <property name="geometry" >
         <rect>
          <x>100</x>
          <y>55</y>
          <width>111</width>
          <height>26</height>
         </rect>
        </property>
        <property name="editable" >
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QComboBox" name="comboMidiRxEnable" >
        <property name="geometry" >
         <rect>
          <x>100</x>
          <y>25</y>
          <width>111</width>
          <height>26</height>
         </rect>
        </property>
        <property name="maximumSize" >
         <size>
          <width>111</width>
          <height>26</height>
         </size>
        </property>
        <property name="editable" >
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_3" >
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>30</y>
          <width>91</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text" >
         <string>MIDI Input:</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Debug Info</string>
     </property>
     <layout class="QHBoxLayout" name="Layout3" >
      <item>
       <widget class="QTableWidget" name="tblDebug" >
        <column>
         <property name="text" >
          <string>Device</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>MIDI 
Type</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Channel</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>MIDI #</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
